src/linker/cli.py

```diff
@@ -138,6 +140,13 @@ def run_slurm_job(
     main = handle_exceptions(
         func=runner.run_container, exceptions_logger=logger, with_debugger=False
     )
+    # Put the implementation_config back to a dictionary
+    reformatted_config = {}
+    if config:
+        for item in config:
+            key, value = item.split("=")
+            reformatted_config[key.strip()] = value.strip()
```

Comment:
This could be a dict comprehension. Not sure if that'd be clearer or harder to read